apps/fabric-website-resources/README.md

```diff
@@ -1,9 +1,9 @@
 # @uifabric/fabric-website-resources
 
-**Local demo app for office-ui-fabric-react controls**
+**Local demo app for @fluentui/react controls**
```

Comment:
hmmm I think #12302 will affect this naming... shouldn't it be changed to @fluentui/react-experimental

Reply:
No this is only for what was formerly OUFR